Doc/library/operator.rst

```diff
@@ -70,6 +70,11 @@ truth tests, identity tests, and boolean operations:
    equivalent to using the :class:`bool` constructor.
 
 
+.. function:: is_none(a)
+
+   Return ``a is None``.  Tests object identity.
```

Comment:
```suggestion
   Return ``a is None``.  Tests object identity.
   
   .. versionadded:: 3.13
```

Also, I think I would put it below the docs for `is_` and `is_not`, since those are more general-purpose functions